store/rootmulti/store.go

```diff
@@ -299,10 +299,7 @@ func (rs *Store) Commit() types.CommitID {
 	version := rs.lastCommitInfo.Version + 1
 	rs.lastCommitInfo = commitStores(version, rs.stores)
 
-	// write CommitInfo to disk only if this version was flushed to disk
-	if rs.pruningOpts.FlushVersion(version) {
```

Comment:
/cc @AdityaSripal since we're reverting to the old behavior, I removed the check but still commit the info. Let me know if committing the info is redundant now.

Reply:
This is correct. From my understanding, the new behavior is to commit every version.

If so, you want to remove the check and still commit